src/utils/viewer/viewer.cpp

```diff
@@ -525,6 +549,106 @@ void Viewer::switchCameraType() {
   }
 }
 
+void Viewer::saveCameraTransformToFile() {
+  if (!Cr::Utility::Directory::exists(saved_transformations_directory)) {
+    Cr::Utility::Directory::mkpath(saved_transformations_directory);
+  }
+
+  // update temporary save
+  lastCameraSave_ = renderCamera_->node().absoluteTransformation();
+  cameraSaved = true;
+
+  // update save in file system
+  saveNodeTransformToFile(renderCamera_->node(),
+                          saved_transformations_directory + "camera." +
+                              getCurrentTimeString() + ".txt");
+}
+
+void Viewer::loadCameraTransformFromFile() {
+  if (!Cr::Utility::Directory::exists(saved_transformations_directory)) {
+    LOG(INFO) << "Cannot find saved transformations directory. Make sure that "
+                 "you've saved a transformation before trying to load one.";
+    return;
+  }
+
+  if (!cameraLoadTimeString_.empty()) {
+    // loading from file system
+    loadNodeTransformFromFile(renderCamera_->node(),
+                              saved_transformations_directory + "camera." +
+                                  cameraLoadTimeString_ + ".txt");
+  } else {
+    // attempting to load from last temporary save
+    LOG(INFO) << "Camera transform file not specified, attempting to load from "
```

Comment:
Start with: `Note:` or `Warning:`

Reply:
I also suggest using `LOG(INFO)`, `LOG(WARNING)`, or `LOG(ERROR)` based on your intention. Users can choose to filter their debug output based on these categories.